src/server/routes/user_reports.ts

```diff
@@ -20,22 +20,32 @@ export default async function handleUserReports(req: Request, res: Response) {
     const [[rawReports], [rawUrlPatterns]] = await Promise.all([
       bq.query({
         query: `
-          SELECT reports.*,
+          SELECT
+            reports.document_id AS uuid,
```

Comment:
nit: we probably don't need `AS uuid` here since `reports.document_id` is used directly?

Reply:
I'm using the `uuid` in the frontend as a key for the iterator. I wanted to keep the diff as small as possible, so I decided to keep the old column names. An alias is essentially free. If we do bigger refactorings, we can drop them, but for now, just re-using the old column names is fine, I'd say.